massdriver-application-aws-ecs-service/security-group.tf

```diff
@@ -0,0 +1,29 @@
+resource "aws_security_group" "service" {
```

Comment:
I can't comment on the `outputs.tf` file but this is looking like a common one if you want to add it.

```
output "id" {
  value = module.application.id
}
```